source/reference/program/mongod.txt

```diff
@@ -156,8 +156,6 @@ Storage Options
 
 .. include:: /includes/option/option-mongod-nojournal.rst
 
-.. include:: /includes/option/option-mongod-journalOptions.rst
```

Comment:
Since it's an existing configuration setting that is being changed in 3.0, I'm hesitant to remove this as some people might have this in use.  Unless there was another reason for removing.